doc/src/content/xdocs/spec.xml

```diff
@@ -1404,10 +1404,11 @@ void initFPTable() {
 }
         </source>
 
-        <p> Readers interested in the mathematics behind this
-        algorithm may want to read <a
-        href="https://www.scribd.com/fb-6001967/d/84795-Crc">this book
-        chapter.</a> (Unlike RFC-1952 and the book chapter, we prepend
+        <p>Readers interested in the mathematics behind this
+          algorithm may want to read
+        <a href="https://books.google.com/books?id=XD9iAwAAQBAJ&amp;pg=PA319"
```

Comment:
If this is meant to link to the specific chapter, it doesn't work for me!  I do get the Google books reference to the book however.

Reply:
Hah, good old Google, giving different results to different viewers.

Reply:
I'm not really sure what better to do here -- it's not clear that the online PDFs of the book I could find were legitimate and freely licensed, so I guess I can either link to the book and let the interested folks acquire it via their own means, or I can remove the link entirely.

Reply:
> let the interested folks acquire it via their own means

That's likely the best solution!  I'll comment on the JIRA for posterity!